website/server/libs/apiMessages.js

```diff
@@ -0,0 +1,25 @@
+// A map of messages used by the API that don't need to be translated and
```

Comment:
I think this lib has been implemented already somewhere else

Reply:
Yeah, this was cherry picked by accident